shapes/contacts/person.shex

```diff
@@ -0,0 +1,24 @@
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+
+# use sameAs for webId
+# sameAs is reflexive
+
+vcard:Individual {
```

Comment:
Probably good to check what we can use from https://solidsdk.inrupt.net/sdk/userprofile.shex